latch_cli/utils.py

```diff
@@ -74,9 +74,23 @@ def _normalize_remote_path(remote_path: str):
     return remote_path
 
 
-def with_si_suffix(num, suffix="B"):
-    for unit in ["", "K", "M", "G", "T", "P", "E", "Z"]:
+def _si_number_strings(num):
+    for unit in ["", "k", "M", "G", "T", "P", "E", "Z"]:
```

Comment:
why lowercase k

Reply:
SI prefix for "kilo" is lowercase "k" :shrug: